080-Recipes/010-csv-data/010-import-csv-data-UI.mdx

```diff
@@ -0,0 +1,106 @@
+navTitle: Import CSV data
+keywords: ['csv', 'import', 'migrate data']
+description: Import records from a local CSV file into a Xata database table
+slug: recipes/import-data
+published: true
+---
+
+## Overview
+
+A CSV file is a standard text file where data fields are typically divided by commas (,), and saved with the `.csv` extension.
+You can import and work with CSV file data in your Xata database.
+
+**NOTE:** While CSV files are explicity mentioned, Xata supports the import of various file formats, including tab-separated values (TSV), comma-separated values (CSV), delimiter-separated values (DSV), and plain text data. This covers a range of extensions including `.tsv`, `.csv`, `.dsv`, and `.txt` formats.
+
+You can import CSVs using the following methods:
+
+- Import a CSV file using the Xata UI (recommeneded)
+- Import a CSV file using the Xata CLI
+
+### Considerations
+
+- Decide if you are using the Xata UI (Workspace) or CLI to import CSV files. Using the UI is the recommended option, but you can use both to edit records after importing.
+- Ensure your file's accuracy before importing. Certain values may become uneditable after importing. For errors in the imported file, you'll need to delete, edit the CSV file, and try importing again.
+
+## Formatting data
+
+When importing CSV files into Xata, ensure the data aligns with the correct format for the database. You can add additional columns and specify data types after importing data.
+
+### Format types
+
+Choose a format that matches the data you're entering. Xata columns support the following formats:
+
+| Format Type   | Description                                               | Example                           |
+| ------------- | --------------------------------------------------------- | --------------------------------- |
+| String        | Basic textual information                                 | "Value"                           |
+| Text          | Longer pieces of textual content                          | "Hello world and welcome to Xata" |
+| Integer       | Whole numbers without decimal points                      | `42`                              |
+| Float         | Numbers with decimal points                               | `3.14`                            |
+| Boolean       | Represents true or false values                           | `true`                            |
+| Datetime      | Date and time information                                 | 2023-08-30T00:00:00               |
+| Link to Table | Creates relationships between tables                      | Table: Orders                     |
+| Email         | Email addresses                                           | `user@example.com`                |
+| Multiple      | Multiple values from a predefined set separated by commas | \["Red", "Green"]                 |
+| File          | Reference to an external file, attachment, or document    | `file.pdf`                        |
```

Comment:
Not sure about the description column, should / could we link our core docs for each column type (which might also need updating 🧶 ):  /docs/concepts/data-model#string etc.

We don't support file or vector. My vote is to include them in the table and say: "Not supported" to make it clear.